### PR TITLE
Create getting_started.md

### PR DESCRIPTION
**Pull request by dgmccart**
Not yet tested on my machine, but I can make code changes following general feedback.

### CODE REVIEW

File: getting_started.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+
+or with virtualenv:
+
+```shell
+$ python -m venv venv
+$ . ./venv/bin/activate # or on Windows: .\venv\Scripts\Activate.bat or .\venv\Scripts\Activate.ps1
+(venv) $ python -m pip install acquire-imaging
+```
+
+Once you have Acquire installed, simply as call `import acquire` in your script, notebook, or module to start utilizing the package.
```
Comment (aliddell) [inline getting_started.md:32]:
```suggestion
Once you have Acquire installed, simply call `import acquire` in your script, notebook, or module to start utilizing the package.
```

File: getting_started.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+```python
+import acquire
+```
+
+## Supported Cameras and File Formats
+
+Acquire supports the following cameras (currently only on Windows):
+
+- [Hamamatsu Orca Fusion BT (C15440-20UP)](https://www.hamamatsu.com/eu/en/product/cameras/cmos-cameras/C15440-20UP.html)
+- [Vieworks VC-151MX-M6H00](https://www.visionsystech.com/products/cameras/vieworks-vc-151mx-sony-imx411-sensor-ultra-high-resolution-cmos-camera-151-mp)
```
Comment (aliddell) [inline getting_started.md:43]:
```suggestion
- [Vieworks VC-151MX-M6H00](https://www.visionsystech.com/products/cameras/vieworks-vc-151mx-sony-imx411-sensor-ultra-high-resolution-cmos-camera-151-mp)
- [FLIR Blackfly USB3 (BFLY-U3-23S6M-C)](https://www.flir.com/products/blackfly-usb3/?model=BFLY-U3-23S6M-C&vertical=machine+vision&segment=iis)
- [FLIR Oryx 10GigE (ORX-10GS-51S5M-C)](https://www.flir.com/products/oryx-10gige/?model=ORX-10GS-51S5M-C&vertical=machine+vision&segment=iis)
```

These should be coming very soon, if not today.

File: getting_started.md [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+import acquire
+
+runtime = acquire.Runtime()
+dm = runtime.device_manager()
+
+for device in dm.devices():
+    print(device)
+```
+The **runtime** is the main entry point in Acquire.
+Through the runtime, you configure your devices, start acquisition, check acquisition status, and terminate acquisition.
```
Comment (aliddell) [inline getting_started.md:86]:
```suggestion
Through the runtime, you configure your devices, start acquisition, check acquisition status, inspect data as it streams from your cameras, and terminate acquisition.
```

File: getting_started.md [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+config.video[1].storage.settings.filename = "output2.zarr"
+
+# where applicable, how large should a chunk file get before opening the next chunk file
+config.video[1].storage.settings.bytes_per_chunk = 64 * 2**20  # 64 MiB chunk sizes
+```
+
+Finally, let's specify how many frames to generate for each camera before stopping our simulated acquisition.
+We also need to register our configuration with the runtime.
+
+If you want to let the runtime just keep acquiring forever, you can set `max_frame_count` to 0.
```
Comment (aliddell) [inline getting_started.md:176]:
```suggestion
If you want to let the runtime just keep acquiring effectively forever, you can set `max_frame_count` to `2**64 - 1`.
```

Setting to 0 just terminates acquisition immediately now.

File: getting_started.md [hunk 5]
```diff
@@ -0,0 +1,253 @@
...
+
+
+```python
+config.video[0].storage.identifier = dm.select(acquire.DeviceKind.Storage, "Zarr")
+
+# what file or directory to write the data to
+config.video[0].storage.settings.filename = "output1.zarr"
+
+# where applicable, how large should a chunk file get before opening the next chunk file
+config.video[0].storage.settings.bytes_per_chunk = 32 * 2**20  # 32 MiB chunk sizes
```
Comment (aliddell) [inline getting_started.md:159]:
```suggestion
config.video[0].storage.settings.chunking.max_bytes_per_chunk = 32 * 2**20  # 32 MiB chunk sizes
```
new API as of v0.1.4

File: getting_started.md [hunk 6]
```diff
@@ -0,0 +1,253 @@
...
+
+
+```python
+config.video[1].storage.identifier = dm.select(acquire.DeviceKind.Storage, "Zarr")
+
+# what file or directory to write the data to
+config.video[1].storage.settings.filename = "output2.zarr"
+
+# where applicable, how large should a chunk file get before opening the next chunk file
+config.video[1].storage.settings.bytes_per_chunk = 64 * 2**20  # 64 MiB chunk sizes
```
Comment (aliddell) [inline getting_started.md:170]:
```suggestion
config.video[1].storage.settings.chunking.max_bytes_per_chunk = 64 * 2**20  # 64 MiB chunk sizes
```

File: getting_started.md [hunk 7]
```diff
@@ -0,0 +1,253 @@
...
+viewer1 = napari.view_image(data1)
+```
+
+```python
+viewer2 = napari.view_image(data2)
+```
+
+## Conclusion
+
+For more examples of using Acquire, check out our [tutorials page](). #ADD LINK
```
Comment (aliddell) [inline getting_started.md:253]: > #ADD LINK

👀

**Review by aliddell — COMMENTED** [review]
Couple of things need to be fixed.

**Comment by melissawm** [comment]
Hi folks, can we merge this one so I can include in the initial website? It would be helpful to have some basic content to check the look and feel 😄  Cheers!

File: getting_started.md (hunk 7 above)
Comment (dgmccart) [reply getting_started.md:255]: I'll add link once we have the tutorials page setup.

**Comment by dgmccart** [comment]
@aliddell Should be ready to merge

File: getting_started.md [hunk 8]
```diff
@@ -93,24 +93,23 @@ We can update the configuration with identifiers from the the runtime's **device
...
 config = runtime.get_configuration()
 ```
 
-Configure the first video stream by setting properties on `config.video[0]` and the second video stream with `config.video[1]`. We'll be using simulated cameras, one generating a radial sine pattern and one generating a random pattern.
-
+Video streams are configured independently. Configure the first video stream by setting properties on `config.video[0]` and the second video stream with `config.video[1]`. We'll be using simulated cameras, one generating a radial sine pattern and one generating a random pattern.
 
 ```python
-config.video[0].camera.identifier = dm.select(acquire.DeviceKind.Camera, "simulated: radial sin")
+config.video[0].camera.identifier = dm.select(acquire.DeviceKind.Camera, "simulated: radial sine")
```
Comment (aliddell) [inline getting_started.md:105]:
```suggestion
config.video[0].camera.identifier = dm.select(acquire.DeviceKind.Camera, "simulated: radial sin")
```
This is actually how it's specified, e.g., [here](https://github.com/acquire-project/acquire-driver-common/blob/main/src/basics.driver.c#L87).

File: getting_started.md [hunk 9]
```diff
@@ -0,0 +1,244 @@
...
+
+# the data type representing each pixel; here we choose unsigned 8-bit integer
+config.video[1].camera.settings.pixel_type = acquire.SampleType.U8
+
+# the shape, in pixels, of the image; width first, then height
+config.video[1].camera.settings.shape = (1280, 720)
+```
+
+Now we'll configure each output, or sink device.
+For both simulaed cameras, we'll be writing to Zarr, a format which supports chunked arrays.
```
Comment (aliddell) [inline getting_started.md:140]:
```suggestion
For both simulated cameras, we'll be writing to Zarr, a format which supports chunked arrays.
```

File: getting_started.md [hunk 10]
```diff
@@ -0,0 +1,244 @@
...
+
+Acquisition happens in a separate thread, so at any point we can check on the status by calling `runtime.get_state()`.
+
+
+```python
+runtime.get_state()
+```
+
+Finally, once we're done acquiring, we call `runtime.stop()`.
+This method will wait until you've reached the maximum number of frames specified in `config.video[0].max_frame_count`.
```
Comment (aliddell) [inline getting_started.md:208]:
```suggestion
This method will wait until you've reached the number of frames specified in `config.video[0].max_frame_count` or `config.video[1].max_frame_count`, whichever is larger.
```

File: getting_started.md [hunk 11]
```diff
@@ -0,0 +1,244 @@
...
+config.video[0].max_frame_count = 100 # collect 100 frames
+config.video[1].max_frame_count = 150 # collect 150 frames
+
+config = runtime.set_configuration(config)
+```
+
+---
+**NOTE**
+
+If you are using a Jupyter notebook for this tutorial, clear output from previous runs with:
```
Comment (aliddell) [inline getting_started.md:179]:
```suggestion
If you run this tutorial multiple times, you can clear output from previous runs with:
```

**Review by aliddell — CHANGES_REQUESTED** [review]

**Review by aliddell — APPROVED** [review]

**Pull request merged**